cloud/k8s/deploy/kontain-deploy/overlays/kkm/cm-install-lib-kkm.yaml

```diff
@@ -56,5 +64,10 @@ data:
       chroot ${ROOT_MOUNT_DIR} ln -sf /opt/kontain/shim/containerd-shim-krun-v2 /usr/bin/containerd-shim-krun-v2
 
       chmod +x ${ROOT_MOUNT_DIR}/kontain-artifacts/kkm.run
```

Comment:
So sometimes you skip chroot and just go to the path ...

Reply:
In the case above, as in the case with the download and untar, yes we use the ${ROOT_MOUNT_DIR} to copy and prep the artifacts.  When dealing with paths that directly affect the host, we use the chroot.

Reply:
But, will revisit use of chroot vs normal next time I revisit this to keep it consistent.